.circleci/config.yml

```diff
@@ -17,20 +17,13 @@ jobs:
     steps:
     - prometheus/setup_environment
     - run:
-        command: make style unused vet build
+        command: make all
         environment:
           # By default Go uses GOMAXPROCS but a Circle CI executor has many
           # cores (> 30) while the CPU and RAM resources are throttled. If we
           # don't limit this to the number of allocated cores, the job is
           # likely to get OOMed and killed.
           GOOPTS: "-p 2"
-    - run: git diff --exit-code
```

Comment:
@geekodour 	any idea was was this here?

Reply:
I used them here because they were used in the prometheus/prometheus `.config.yml` 
I think it's a check to verify that no new modules were added and no other files get changed when running `make build`.

Reply:
I don't see it anymore there so shall we leave it out as well?
https://github.com/prometheus/prometheus/blob/master/.circleci/config.yml

Reply:
yes i think we can remove it as well.